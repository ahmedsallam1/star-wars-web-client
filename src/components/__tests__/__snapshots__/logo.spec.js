import { mount } from '@vue/test-utils'
import Logo from '../../logo'

describe('Logo', () => {
	test('renders correctly', () => {
		const wrapper = mount(Logo)
	  	expect(wrapper.element).toMatchSnapshot()
	})
})