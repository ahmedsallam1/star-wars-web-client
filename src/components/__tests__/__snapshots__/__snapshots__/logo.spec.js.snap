// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<div>
  <div
    id="logo"
  >
    <img
      src="../assets/img/logo.svg"
    />
  </div>
</div>
`;
